### PR TITLE
Added additional information about updating customer's store credit balance

### PR DESCRIPTION
**Pull request by DmitryTsymbal**
## Purpose of this pull request

Added additional information about updating customer's store credit balance

## Affected documentation pages

https://docs.magento.com/user-guide/customers/store-credit-using.html

whatsnew
Updated the [Applying Store Credit](https://docs.magento.com/user-guide/customers/store-credit-using.html) topic to include information about updating a customer's store credit balance.

### CODE REVIEW

File: src/customers/store-credit-using.md [hunk 1]
```diff
@@ -23,6 +23,28 @@ _Customer credit balance and history_
...
+
+1. In the _Action_ column, click **Edit**.
+
+1. In the left panel, choose **Store Credit**.
+
+1. Choose the website (storefront) for which the balance will be associated.
+
+1. For **Update Balance**, enter the new value.
+
+1. To notify customer about balance updating check **Notify Customer by Email** checkbox and choose the store view from **Send Email Notification From the Following Store View**.
```
Comment (leslietilling) [inline src/customers/store-credit-using.md:40]:
```suggestion
1. To notify the customer about the balance update, select the **Notify Customer by Email** checkbox and choose the store view from **Send Email Notification From the Following Store View**.
```

File: src/customers/store-credit-using.md [hunk 2]
```diff
@@ -23,6 +23,28 @@ _Customer credit balance and history_
...
+
+1. Enter the **Comment**
```
Comment (leslietilling) [inline src/customers/store-credit-using.md:42]:
```suggestion
1. Enter a **Comment** about the change.
```

File: src/customers/store-credit-using.md [hunk 3]
```diff
@@ -23,6 +23,28 @@ _Customer credit balance and history_
...
+
+1. When updates are complete, click <span class="btn">Save and Continue Edit</span> or <span class="btn">Save Customer</span>.
+
+Updated Balance entity should be displayed in **Balance History**
```
Comment (leslietilling) [inline src/customers/store-credit-using.md:46]:
```suggestion
Updated Balance entity should be displayed in **Balance History**.
```

File: src/customers/store-credit-using.md [hunk 4]
```diff
@@ -23,6 +23,28 @@ _Customer credit balance and history_
...
+
+1. On the _Admin_ sidebar, go to **Customers** > _Operations_ > **All Customers**.
+
+1. Find the customer in the grid.
+
+1. In the _Action_ column, click **Edit**.
+
+1. In the left panel, choose **Store Credit**.
+
+1. Choose the website (storefront) for which the balance will be associated.
```
Comment (leslietilling) [inline src/customers/store-credit-using.md:36]:
```suggestion
1. Choose the website (storefront) to associate with the balance.
```

**Review by leslietilling — CHANGES_REQUESTED** [review]
Edits for style and clarity

**Review by leslietilling — APPROVED** [review]

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by m2-github-services** [comment]
@leslietilling the Pull Request is successfully imported.

**Pull request merged**